doc/build/html/_sources/introduction.rst.txt

```diff
@@ -0,0 +1,46 @@
+Introduction
+==========================
+
+**The Sheriff** is an exciting game application about an **outlaw** that
+goes out to eat with his **gang friends** in some restaurant. Along the game,
+the player will be able to **organize raids** in any restaurant, **create new gangs** and
+**many more features**!
+
+This game is created by the the company *StartupZ*, leaded by Fran Mangas,
+who brang the business project, and developed by *The Sheriff Team*, a group of Software
+Engineers from the *Evil Corp* consulting company, composed by Tatiana Bagnara,
+Diego Zavaleta, Ricardo Muhamed, Sergi Jimenez and Quim Nadal.
+
+The goal of doing this project is to **learn**, **apply** and later **evaluate** what the *Evil Corp*
+developers learnt in *Diseño Software 1* subject:
+
+1. **Clean code technics**
+
+2. **Creational, structural and behavioral patterns**
+
+3. **SOLID principles.**
+
+**The Sheriff** is an activity is contained in *Diseño Software 1* subject, done in the first semester of the
+*Master en Desarrollo y Arquitectura de Software* degree, in *La Salle, Universitat Ramón
+Llull* university.
+
+Document Contents
+-----------------
+
+In all the other sections basically it is going to be explained the next information:
+
+- **Functionality - The Sheriff usage**: How to run the application, how to run the code static analysis,
```

Comment:
The document `usage.rst` was aiming exactly this.

Reply:
I know, and the introduction introduces that we are going to talk about this later!